### DISCUSSION TITLE
ANEs and Swift on iOS

### DISCUSSION THREAD

**Discussion by pol2095** [airsdk/Adobe-Runtime-Support#2664] [Q&A] [answered]
Hello,

is-it possible to use two ANE using Swift on iOS ?
I tried, it seems not working.

Thanks

**Comment by marchbold** [comment, marked as answer]
Yes... What isn't working?

**Reply by pol2095** [reply]
when I instanciate the first ANE, it work, when I instantiate the second ANE, the app crash.
or when I instanciate the second ANE, it work, when I instantiate the first ANE, the app crash.

**Reply by pol2095** [reply]
It work, I have a variable using the same name in my 2 static libraries, thanks.

**Reply by marchbold** [reply]
Yep, that would do it :) Same for objc global functions and variables.
